src/OpenGL/Extensions/Silk.NET.OpenGL.Extensions.ImGui/README.txt

```diff
@@ -1,3 +1,11 @@
-﻿Adapted from https://github.com/NogginBops/ImGui.NET_OpenTK_Sample/ under the provisions of the MIT license,
+﻿Adapted from 
+
+OpenGL 3 reference implementation of ImGui at:
+https://github.com/ocornut/imgui/blob/master/backends/imgui_impl_glfw.cpp
+https://github.com/ocornut/imgui/blob/master/backends/imgui_impl_opengl3.cpp
+https://github.com/ocornut/imgui/blob/master/examples/example_glfw_opengl3/main.cpp
+
+OpenTK at:
+https://github.com/NogginBops/ImGui.NET_OpenTK_Sample/ under the provisions of the MIT license,
 which itself was adapted from Veldrid.ImGui at https://github.com/mellinoe/veldrid/ under the provisions of the MIT
 license.
```

Comment:
Do we need this anymore? We only need this if you used this as a reference in this rewrite.

Reply:
Removed. I did look at it, but it in the end, not used.